entities.json

```diff
@@ -1,4 +1,5 @@
 {
+    "": "Automatically generated, do not edit. Refer to entities.mjs for instructions.",
```

Comment:
I think this file is only accessed by Perl now, so if we wanted to, we could just generate a Perl module instead (so it doesn't have to read/parse JSON). But I kept it simple for now...